spec/paperclip/attachment_registry_spec.rb

```diff
@@ -77,7 +106,9 @@
   context '.clear' do
     it 'removes all of the existing attachment definitions' do
       foo = Class.new
-      Paperclip::AttachmentRegistry.register(foo, :greeter, { ciao: 'greeting' })
+      Paperclip::AttachmentRegistry.register(foo,
+                                             :greeter,
+                                             { ciao: 'greeting' })
```

Comment:
Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.